src/builder/payload.rs

```diff
@@ -1,13 +1,37 @@
-static PROTO: &str = "__proto__";
-
 pub fn get() -> Vec<String> {
+	let mut payload = vec![];
+
+	let object = get_object();
+	let pointer = get_pointer();
+
+	payload.extend(object);
+	payload.extend(pointer);
+
+	payload
```

Comment:
```suggestion
	vec![ get_object(), get_pointer() ]
```

Reply:
It says
```
expected struct `std::string::String`, found struct `Vec`
```

Reply:
Ah my bad, missed that get_* return vecs as well, then that's fine, only suggestion would be:

```rust
let mut payload = get_object();
payload.extend(get_pointer());
```

Reply:
@mattsse, I ended up with `concat`. :smile:

Reply:
yeah, there way too many ways to write this and all probably end up the same

Reply:
Can you please review this one more?